src/display/font_loader.js

```diff
@@ -71,19 +80,54 @@ class FontLoader {
     }
   }
 
+  async loadSystemFont(info) {
+    assert(
+      !this.disableFontFace,
+      "loadSystemFont shouldn't be called when `disableFontFace` is set."
+    );
+
+    const { loadedName, src, style } = info;
+    if (this.#systemFonts.has(src)) {
+      return;
+    }
+    this.#systemFonts.add(src);
+    if (this.isFontLoadingAPISupported) {
+      const fontFace = new FontFace(loadedName, src, style);
+      this.addNativeFontFace(fontFace);
+      try {
+        await fontFace.load();
+      } catch {
+        warn(
+          `Cannot load system font: ${loadedName} for style ${style.style} and weight ${style.weight}.`
+        );
+        this.removeNativeFontFace(fontFace);
+      }
+      return;
+    }
+
+    unreachable(
+      "Not implemented: loadSystemFont without the Font Loading API."
+    );
+  }
+
   async bind(font) {
     // Add the font to the DOM only once; skip if the font is already loaded.
-    if (font.attached || font.missingFile) {
+    if (font.attached || (font.missingFile && !font.systemFontInfo)) {
       return;
     }
     font.attached = true;
 
+    if (font.systemFontInfo) {
+      await this.loadSystemFont(font.systemFontInfo);
+      return;
+    }
+
     if (this.isFontLoadingAPISupported) {
       const nativeFontFace = font.createNativeFontFace();
       if (nativeFontFace) {
         this.addNativeFontFace(nativeFontFace);
         try {
-          await nativeFontFace.loaded;
+          await nativeFontFace.load();
```

Comment:
Given the size/scope of this patch, I'd probably suggest that we don't add any unnecessary risk by not making this "unrelated" change here.

Reply:
Looking at https://developer.mozilla.org/en-US/docs/Web/API/FontFace/load it contains the following:
> If the source for the font face was specified as binary data, or the font [status](https://developer.mozilla.org/en-US/docs/Web/API/FontFace/status) property of the font face is anything other than unloaded, then this method does nothing.